desktop/libs/aws/src/aws/client.py

```diff
@@ -45,7 +45,7 @@ def get_credential_provider(identifier, user):
 def _make_client(identifier, user):
   client_conf = aws_conf.AWS_ACCOUNTS[identifier] if identifier in aws_conf.AWS_ACCOUNTS else None
 
-  if RAZ.IS_ENABLED.get() and bool(get_raz_s3_default_bucket()) and not aws_conf.IS_SELF_SIGNING_ENABLED.get():
+  if not aws_conf.IS_SELF_SIGNING_ENABLED.get():
```

Comment:
@romainr  here do we need to keep `RAZ.IS_ENABLED.get()` or no?

Reply:
Yes

Reply:
Looks like we are breaking a test here since raz is enabled it removed.

Reply:
Added it back.